### DISCUSSION TITLE
Create daily / weekly summary about activities for every member via e-mail

### DISCUSSION THREAD

**Discussion by BRogers80** [outline/outline#3259] [Ideas]
Dear team,

we use outline mainly for our internal knowledgebase. It would be great if there ist the possibility that every employee will receive a summary email about new and updated articels on a daily or weekly base so everybody gets in touch with the new content.

We have several people using the wiki only a few times per month. With this feature they will be automatically up to date if new articles are online and do not bother other colleagues with their questions.

Thank you

Flo

**Comment by tommoor** [comment]
This is a great idea, would love to include this feature

**Comment by roovyshapiro** [comment]
I just wanted to comment to say that I implemented my own workaround for this.
I use the List all documents API (https://www.getoutline.com/developers#tag/Documents/paths/~1documents.list/post) with
`   
json= {
        "sort": "updatedAt",
        "direction": "DESC",
    }
`
That gives me all recent document updates. I store all these "contributions" into a DB. I then also send them out to our internal chat app (Flock), I can send them via email, I can do some analytics on who's making the most contributions in a given time frame etc.
It's not perfect, but it does what I need.

**Reply by aspoet** [reply]
That is a very nice solution but it would be awesome to have this for the cloud hosted version as well - several team members already asked independently if this was possible .
